package.json

```diff
@@ -31,6 +31,7 @@
     "@babel/core": "^7.12.10",
     "@babel/preset-env": "^7.12.11",
     "@babel/register": "^7.12.10",
+    "@rollup/plugin-buble": "^0.21.3",
```

Comment:
Just flagging that babel is probably a better long-term choice than buble, but this is probably fine for several years and a lot less work then setting up all of babel.